akka-docs/src/main/paradox/typed/persistence.md

```diff
@@ -242,16 +242,16 @@ Java
 
 ## Changing Behavior
 
-After processing a message, plain typed actors are able to return the `Behavior` that is used
+After processing a message, actors are able to return the `Behavior` that is used
 for next message.
 
-As you can see in the above examples this is not supported by typed persistent actors. Instead, the state is
+As you can see in the above examples this is not supported by persistent actors. Instead, the state is
 returned by `eventHandler`. The reason a new behavior can't be returned is that behavior is part of the actor's
 state and must also carefully be reconstructed during recovery. If it would have been supported it would mean
 that the behavior must be restored when replaying events and also encoded in the state anyway when snapshots are used.
-That would be very prone to mistakes and thus not allowed in Typed Persistence.
+That would be very prone to mistakes and thus not allowed in Akka Persistence.
 
-For basic actors you can use the same set of command handlers independent of what state the entity is in,
+For basic actors you can use the same set of command handlers independpedcanent of what state the entity is in,
```

Comment:
Not quite sure about this word ;)

Reply:
Sometimes I make things up to look smart. Fixing ;)